frontend/javascripts/oxalis/model/accessors/flycam_accessor.js

```diff
@@ -294,9 +294,9 @@ export function getValidTaskZoomRange(
   respectRestriction: boolean = false,
 ): [number, number] {
   const defaultRange = [userSettings.zoom.minimum, Infinity];
-  const { allowedMagnifications } = state.tracing.restrictions;
+  const { resolutionRestrictions } = state.tracing.restrictions;
 
-  if (!respectRestriction || !allowedMagnifications || !allowedMagnifications.shouldRestrict) {
+  if (!respectRestriction || !resolutionRestrictions) {
```

Comment:
Isn't the new format

```
  +resolutionRestrictions: {
    min?: number,
    max?: number,
  },
```

? Then, this would need to check whether min and max are both null?

Reply:
The way I understand this the code below does that for min and max individually, and essentially results in the default range if both are null. Am I misreading that? In fact we could probably even remove the `!resolutionRestrictions` check here too, as it should now always be present. Do you agree?

Reply:
> In fact we could probably even remove the !resolutionRestrictions check here too, as it should now always be present.

Yes, that was the point which confused me initially :)

> The way I understand this the code below does that for min and max individually, and essentially results in the default range if both are null. 

Yes, that should be correct!